src/constants/Constants.js

```diff
@@ -1,9 +1,14 @@
+
 const BASE_URL = 'https://phlox-ah-staging.herokuapp.com';
+
 export const articleConstant = {
   ALL_ARTICLES_URL: `${BASE_URL}/api/v1/articles/feed`,
   CREATE_ARTICLES_URL: `${BASE_URL}/api/v1/articles`,
   UPDATE_ARTICLE_URL: `${BASE_URL}/api/v1/articles`,
+  MY_ARTICLES_URL: `${BASE_URL}/api/v1/myarticles`,
   VIEW_ARTICLE_URL: `${BASE_URL}/api/v1/articles`,
+  DELETE_ARTICLE_URL: `${BASE_URL}/api/v1/articles`,
+  SINGLE_ARTICLE_URL: `${BASE_URL}/api/v1/articles`,
```

Comment:
I would suggest that you raise a follow up PR to reduce the variable duplication in this file and other files that need to reflect the change.